apps/admin/sbox/00/kustomization.yaml

```diff
@@ -5,7 +5,9 @@ resources:
   - ../../secrets-csi-driver-crds
   - ../../secrets-csi-driver
   - ../../traefik2
+  - ../../aad-pod-id/mi
 
 patchesStrategicMerge:
   - ../../traefik2/sbox/00.yaml
   - ../../traefik2/sbox/secret-provider.yaml
+  - ../../aad-pod-id/mi/sbox/azure-identity-patch.yaml
```

Comment:
I think it has to be patches give the format in apps/admin/aad-pod-id/mi/sbox/azure-identity-patch.yaml. Instead you can just change the file to this format https://github.com/hmcts/cnp-flux-config/blob/master/apps/adoption/identity/ithc.yaml

Reply:
my bad :) cheers Praveen